### PR TITLE
Made broker actor obsolete

### PR DESCRIPTION
**Pull request by loekd**

### CODE REVIEW

**Comment by loekd** [comment]
I request a review.

**Review by danadesrosiers — APPROVED** [review]

**Comment by danadesrosiers** [comment]
A note in the README warning potential users that the BrokerActor/BrokerRelayActor is being phased out would be nice.

**Review by danadesrosiers — CHANGES_REQUESTED** [review]
There are more references in SubscriberServiceExtensions and SubscriberActorExtensions that should be marked obsolete.

**Comment by loekd** [comment]
@danadesrosiers thanks, processed the feedback and merged.

File: ServiceFabric.PubSubActors/SubscriberServices/SubscriberServiceExtensions.cs [hunk 1]
```diff
@@ -180,7 +188,6 @@ public static Task UnregisterMessageTypeWithRelayBrokerAsync(this StatelessServi
         /// Registers this Service as a subscriber for messages of type <paramref name="messageType"/> with the <see cref="BrokerService"/>.
         /// </summary>
         /// <returns></returns>
-        [Obsolete("Use ServiceFabric.PubSubActors.Helpers.SubscriberServiceHelper for testability")]
```
Comment (danadesrosiers) [inline ServiceFabric.PubSubActors/SubscriberServices/SubscriberServiceExtensions.cs:183]: Did you mean to remove the obsolete attribute from these methods?

**Pull request merged**